### PR TITLE
Add Page for Polkadot-JS

### LINKED ISSUE #2441: Add Further Content for Polkadot.js
A page dedicated to PolkadotJS Apps

### PR DESCRIPTION
**Pull request by salmad3**
Adding this so we have something on the Wiki.

- Adds basic information for Polkadot-JS on a dedicated page. 
- Will be revisited/updated as part reviewal for wiki revamp.

Most of the features that can be done on the Apps UI are described on other pages. If we want to include a basic breakdown of the UI we can do so, or at least the key parts of it. Same with the extension and JS library. Would also be open to having this page updated with screenshots as part of https://github.com/w3f/polkadot-wiki/issues/2390

Resolves #2441

### CODE REVIEW

**Comment by salmad3** [comment]
@DrW3RK @emresurmeli input from both is encouraged

File: docs/learn/learn-polkadotjs.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+The Polkadot-JS UI is a hosted application that loads in your browser. The UI has a standard DNS hosted
+version, which always has the latest features, and an IPFS version that is less frequently updated but 
+is more decentralized. This is also often referred to as Polkadot-JS Apps, or the Apps UI.
+
+Polkadot-JS Apps has many capabilities that go beyond basic wallet functions such as account creation 
+and sending or receiving transactions. 
+
+### [Polkadot-JS Extension](https://polkadot.js.org/extension/)
+
+The extension is a simple proof-of-concept for managing accounts in a browser extension and allowing
```
Comment (emresurmeli) [inline docs/learn/learn-polkadotjs.md]: I'm not sure proof of concept is the right explanation here, but I might be wrong

File: docs/learn/learn-polkadotjs.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+### [Polkadot-JS API](github.com/polkadot-js/api)
+
+A JavaScript API allowing for programs to interface with the functionality of Polkadot.
+
+## Polkadot-JS Apps
+
+Polkadot-JS Apps is the native Polkadot/Kusama/Substrate UI for interacting with a node, allowing access to 
+all features available on Substrate chains.
+
```
Comment (emresurmeli) [inline docs/learn/learn-polkadotjs.md:52]: I would add that it also allows you to interface with a local node, which is useful for developers

File: docs/learn/learn-polkadotjs.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Among other things, Polkadot-JS Apps allows a user to:
+
+- [Staking](learn-staking.md) 
+- [Governance](learn-governance.md)
+- [Crowdloans](learn-crowdloans.md)
+- [Auctions](learn-auction.md)
+- Query chain metadata
+- Query on-chain data using RPC calls
+
```
Comment (emresurmeli) [inline docs/learn/learn-polkadotjs.md:64]: I would add that it allows you to call extrinsics using your accounts

File: docs/learn/learn-polkadotjs.md (hunk 1 above)
Comment (salmad3) [reply docs/learn/learn-polkadotjs.md]: Addressed.

File: docs/learn/learn-polkadotjs.md (hunk 2 above)
Comment (salmad3) [reply docs/learn/learn-polkadotjs.md:52]: Updated.

File: docs/learn/learn-polkadotjs.md (hunk 3 above)
Comment (salmad3) [reply docs/learn/learn-polkadotjs.md:64]: Updated.

File: docs/learn/learn-polkadotjs.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+For more user-friendly wallets, check out the supported and treasury-funded wallets on the 
+[Wallets Page](../build/build-wallets.md)
+
+## Resources
+
+### Beginner's Guide to Polkadot-JS
+
+- [Introduction to Polkadot-JS](https://www.youtube.com/watch?v=4EQqwGFV1D8)
+- [Create an account using Polkadot-JS](https://www.youtube.com/watch?v=sy7lvAqyzkY)
+
```
Comment (profradha) [inline docs/learn/learn-polkadotjs.md:101]:
```suggestion
[Network Explorer on Polkadot-JS UI](https://www.youtube.com/watch?v=g4b4IWR6OrE)

```

**Review by DrW3RK — APPROVED** [review]
Looks good to me. Just added another video made on Polkadot JS UI

**Pull request merged**